dev/com.ibm.ws.webcontainer.servlet.4.0/src/com/ibm/ws/webcontainer40/srt/SRTServletRequest40.java

```diff
@@ -186,8 +186,10 @@ public HttpServletMapping getCurrentHttpServletMapping(WebAppDispatcherContext40
                     returnMapping = new HttpServletMappingImpl(MappingMatch.EXTENSION, matchValue, pattern, servletName);
                     break;
                 case PATH:
```

Comment:
Hey Phu can you just add a comment here so we know in the future why we're only changing the matchValue when there isn't a slashStarMapping?